src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -421,10 +498,10 @@ export class TelemetryHelper {
     }
 
     public resetClientComponentLatencyTime() {
-        this.invokeSuggestionStartTime = 0
-        this.sdkApiCallStartTime = 0
+        CodeWhispererSessionStates.instance.invokeSuggestionStartTime = 0
+        CodeWhispererSessionStates.instance.sdkApiCallStartTime = 0
         this.sdkApiCallEndTime = 0
-        this.fetchCredentialStartTime = 0
```

Comment:
Thanks for simplifying the states. Any chance to move `sdkApiCallEndTime = 0` inside `CodeWhispererSessionStates.instance` ?

Reply:
Yea definitely, simple not a required change for this PR.